config/app/routes.yaml

```diff
@@ -0,0 +1,3 @@
+ngsite_view_payload:
```

Comment:
Can you explain a bit why these routes are used for?

Ideally, this should not be located in media-site, but in netgen/site-bundle.